mne/cov.py

```diff
@@ -997,8 +998,9 @@ def _compute_covariance_auto(data, method, info, method_params, cv,
     cov_methods = [c.__name__ if callable(c) else c for c in method]
     runtime_infos, covs = list(runtime_infos), list(covs)
     my_zip = zip(cov_methods, runtime_infos, logliks, covs, estimators)
-    for this_method, runtime_info, loglik, data, est in my_zip:
-        out[this_method] = {'loglik': loglik, 'data': data, 'estimator': est}
+    for this_method, runtime_info, loglik, cov_data, est in my_zip:
```

Comment:
why the variable name change? if it only lives in this scope. No need for this diff, change back both lines.